RELEASENOTES.md

```diff
@@ -1 +1 @@
-
+Accept archive paths when using GitHub storage.
```

Comment:
Since this change only applies to `gh gei` it's better to be specific:

```suggestion
`gh gei migrate-repo` now allows using either `--ghes-api-url` or archive paths (`--git-archive-path` and `--metadata-archive-path`) when using GitHub owned storage.
```

Reply:
Updated with cd696c3a718e2a38c9813dfb1fbfebf054be0898!